Content.Shared/Inventory/InventorySystem.Equip.cs

```diff
@@ -423,17 +442,27 @@ public bool TryUnequip(
             return false;
         }
 
+        if (!_containerSystem.Remove(removedItem.Value, slotContainer, force: force, reparent: reparent))
+            return false;
```

Comment:
Doesn't moving this change the logic of the recursion?
Before it always went through all of the slots and called Remove on each one.
Now if Remove fails, then the recursion won't start.
HOWEVER this might actually be the correct way it should be. If we try to remove the jumpsuit, but fail for some reason (there is something like an UnremovableComponent), then the PDA or pocket items should not be dropped either.
I just want your confirmation for a sanity check.